theme/static/js/hs-vue/manage-access-app.js

```diff
@@ -51,10 +53,16 @@ let manageAccessApp = new Vue({
 
             let accessStr = "Private";
             if (newAccess.isPublic && newAccess.isDiscoverable) {
-                accessStr = "Public"
+                accessStr = "Public";
+            }
+            else if (!newAccess.isPublic && newAccess.isDiscoverable && newAccess.isPrivateLinkSharing) {
+                accessStr = "Discoverable (Accessible via direct link sharing)";
             }
             else if (!newAccess.isPublic && newAccess.isDiscoverable) {
-                accessStr = "Discoverable"
+                accessStr = "Discoverable";
```

Comment:
Again, ```newAccess.isPublic``` is implicit in the else condition and can be removed.

Reply:
fixed